public/app/plugins/datasource/mysql/CheatSheet.tsx

```diff
@@ -48,10 +48,10 @@ export function CheatSheet() {
         </li>
         <li>
           $__timeGroupAlias(column,&apos;5m&apos;) -&gt; cast(cast(UNIX_TIMESTAMP(column)/(300) as signed)*300 as
-          signed) AS &quote;time&quote;
+          signed) AS &quot;time&quot;
```

Comment:
Is there a reason we don't just use plain quote characters here instead?

Reply:
We have this rule in place https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md